image/php-fpm/www.conf

```diff
@@ -19,7 +19,7 @@
 ; Unix user/group of processes
 ; Note: The user is mandatory. If the group is not set, the default user's group
 ;       will be used.
-user = root
+user = www-data
```

Comment:
Same as previous.  Let's make sure this works when bind-mounted on OSX and/or Windows.